app/public/js/imagebase.js

```diff
@@ -48,21 +48,15 @@ $(function() {
 			propedit.append(arkdiv);
 			SALSAH.ApiGet('resourcesv2', resource.resdata.res_id, {resinfo: true, reqtype: 'context'}, function(datav2) {
 				if (datav2['knora-api:versionArkUrl'] && datav2['knora-api:versionArkUrl']['@value']) {
-					arkdiv.append(
-								$('<em>')
-									.addClass('propedit label')
-									.append('Persistent ID (')
-									.append(
-										$('<a>').attr({href: 'https://en.wikipedia.org/wiki/Archival_Resource_Key'}).append('ARK'))
-									.append('): ')
-							.append($('<img>').attr({src: SITE_URL + '/app/icons/16x16/attachment.png', title: 'Copy to clipboard'}).on('click', function() {
-								$(this).next('.clipit').select();
-								document.execCommand('copy');
-							})
-							.dragndrop('makeDraggable', 'HANDLE_ID', {handle_id: datav2['knora-api:versionArkUrl']['@value']}))
-							.append($('<input>')
-								.attr({type:'text', readOnly: true, size: '64'})
-								.addClass('clipit').val(datav2['knora-api:versionArkUrl']['@value'])));
+					arkdiv.clipurl({
```

Comment:
@lrosenth I replaced this code with the new plugin. Could you please check?

I did not understand the purpose of dragndrop. Is the idea to use the ARK ID for interlinking resources?

Reply:
DOn't we just use resource IRIs to link resources?